app/javascript/components/VerticalNav/VerticalNav.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { GridContainer, Header, NavDropDownButton, Menu, PrimaryNav } from '@trussworks/react-uswds';
+
+import './VerticalNav.css';
+
+export const VerticalNav = () => {
+  const [isOpen, setIsOpen] = useState([false, false]);
+  const [isOpen2, setIsOpen2] = useState([false, false]);
+
+  const onToggle = (
+    index: number,
+    setIsOpen: React.Dispatch<React.SetStateAction<boolean[]>>
+  ): void => {
+    setIsOpen((prevIsOpen) => {
+      const newIsOpen = [false, false];
+      newIsOpen[index] = !prevIsOpen[index];
```

Comment:
Right now the `index` is always `0`. Is there a reason `isOpen` has two booleans? I don't see the second boolean being used.

Reply:
this has been fixed.